test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -205,8 +207,9 @@ void doBlockchainTestNoLog(json_spirit::mValue& _v, bool _fillin)
 				tests[newtestname] = jObj;
 			}
 
-			//delete source test from the json
-			_v.get_obj().erase(_v.get_obj().find(testname));
```

Comment:
I'm only suggesting a pattern I know. Using iterators is always better, because they are the quickest way. Remove list is another data structure and you have to do the find again.